lib/generators/sorcery/templates/initializer.rb

```diff
@@ -523,6 +523,43 @@
     # Default: `:uid`
     #
     # user.provider_uid_attribute_name =
+
+    # -- jwt_auth --
+
+    # Parameters passed for generating payload part of token
+    # Default: [:id]
+    #
+    # config.jwt_user_params =
+
+    # Payload to set like expiration time
+    # See https://github.com/jwt/ruby-jwt what claims can be used
+    # Default: `{}`
+    # config.jwt_payload = 
+
+    # Header name which will parsed
+    # Default: `Authorization`
+    #
+    # config.jwt_headers_key =
+
+    # Key on which returned user data
+    # Default: :user_data
+    #
+    # config.jwt_user_data_key =
+
+    # Key on which returned token
+    # Default: :auth_token
+    #
+    # config.jwt_auth_token_key =
+
+    # A flag that specifies whether to perform a database query to set the current_user
+    # Default: true
+    #
+    # config.jwt_set_user =
```

Comment:
Do we need this flag? Is there ever a case when `current_user` should not be an instance of a user model?